Adyen/Utilities/PhoneExtensionsRepository.swift

```diff
@@ -46,6 +49,8 @@ public enum PhoneNumberPaymentMethod {
                     "KR"]
         case .mbWay:
             return ["PT", "ES"]
+        case .generic:
+            return Array(allCountriesPhoneExtensions.keys).sorted()
```

Comment:
No restrictions whatsoever?

Reply:
Yeah, I asked the designers, still waiting for an answer :-) .

Reply:
they said its okay for now